[CS] Code Style fixes for administrator/components/com_menus/

photodude opened the pull request:
Pull Request for Issue code style fixes

### Summary of Changes
- Blank line found at end of control structure
- Blank line found at start of control structure
- No blank line found after control structure
- Expected "array|boolean" but found "array|bool" for function return type
- Expected "integer" but found "int" for function return type
- Closing parenthesis of a multi-line function call must be on a line by itself
- Multi-line function call not indented correctly
- Expected 1 newline after opening brace
- Tabs must be used to indent lines; spaces are not allowed

[Automatically fixed with Joomla code standards 2.0.0 PHPCS2-alpha2 fixers](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2)

None of the manual only fixes have been applied

### Testing Instructions
Merge by code review

### Expected result
code style has been applied as listed above, old code style check on drone does not error.

### Actual result
code style had not been applied. [Autofixers from the Joomla code standards 2.0.0 PHPCS2 alpha2](https://github.com/joomla/coding-standards/releases/tag/2.0.0-alpha2) were used to implement fixable code style

### Documentation Changes Required
none

photodude committed:
PHPCS2-autofixers

- Blank line found at end of control structure
- Blank line found at start of control structure
- No blank line found after control structure
- Expected "array|boolean" but found "array|bool" for function return
type
- Expected "integer" but found "int" for function return type
- Closing parenthesis of a multi-line function call must be on a line by
itself
- Multi-line function call not indented correctly
- Expected 1 newline after opening brace
- Tabs must be used to indent lines; spaces are not allowed

Quy commented on administrator/components/com_menus/controllers/items.php:
```diff
@@ -174,11 +174,11 @@ public function setDefault()
-						'index.php?option=' . $this->option . '&view=' . $this->view_list
+			JRoute::_(
+				'index.php?option=' . $this->option . '&view=' . $this->view_list
 						. '&menutype=' . $app->getUserState('com_menus.items.menutype'), false
```
Fix indentation

photodude replied:
What are you expecting for indentation?
This is what the autofixer says is correct for this type of multiline based on the applied rule from PEAR.Functions.FunctionCallSignature.Indent

Quy replied:
Line 179 to have same indentation as line 178.

photodude replied:
Agreed, not sure why the fixer didn't address that. 

This might be a case where we'll have to manually adjust.

Quy commented on administrator/components/com_menus/models/items.php:
```diff
@@ -469,7 +469,7 @@ protected function preprocessForm(JForm $form, $data, $group = 'content')
 	 * @param   string  $menuType  The menutype identifier for the menu
 	 * @param   bool    $check     Flag whether to perform check against ACL as well as existence
```
boolean?

photodude replied:
Yes, not sure why the autofixer didn't catch and fix this.
I'll look into it.

photodude replied:
Looks like there may be an issue with our custom sniff Joomla.Commenting.FunctionComment.InvalidReturn

I'll be looking to see if there was a change in the source file it's based on that we need to bring over.

photodude replied:
Corrected manually, not sure why it was missed.

photodude committed:
Manual suggested corrections

Auto fixers didn't fix these items correctly, manually adjusting for PR
review.
